### PR TITLE
pep8 adjustments

### PR DESCRIPTION
**Pull request by sepulworld**

### CODE REVIEW

**Comment by sepulworld** [comment]
Also, I was thinking we should do s3.py functions in a Class like secretsmanaer.py So it can be an object. Thoughts?

File: aws_python_utils/s3.py [hunk 1]
```diff
@@ -3,40 +3,37 @@
...
-"""
-def get_bucket_and_key_from_s3_path(s3_path):
-    res = re.search("s3://(.*?)/(.*)", s3_path)
-    if not res:
-        # try without s3 protocol
-        res = re.search("(.*?)/(.*)", s3_path)
+
+class AwsS3():
+    """AWS Secret Manager class"""
+
```
Comment (ctimmins) [inline aws_python_utils/s3.py]: define `__init__(self, *args)`  constructor to take params for credential configuration for s3 client

Comment (sepulworld) [reply aws_python_utils/s3.py]: Good call. Added to both s3 and secretsmanager classes

**Comment by sepulworld** [comment]
Im going to merge this and then start a new PR that will have moto mocks tests for us to use!

**Pull request merged**